speakeasy/winenv/api/usermode/ntdll.py

```diff
@@ -333,4 +333,73 @@ def LdrAccessResource(self, emu, argv, ctx={}):
         self.mem_write(Resource, offset.to_bytes(4, 'little'))
         
         return 0
+           
+    @apihook('wcsstr', argc=2,conv=e_arch.CALL_CONV_CDECL)
+    def wcsstr(self, emu, argv, ctx={}):
+        """
+        wchar_t *wcsstr(
+            const wchar_t *str,
+            const wchar_t *strSearch
+        );
+        """
+        #Copied from msvcrt
```

Comment:
rather than copying the implementation, could we reuse the one in msvcrt somehow?